ChangeLog.md

```diff
@@ -27,12 +27,17 @@ See docs/process.md for more on how version tagging works.
 - C++17 is now the default version of the C++ standard used by the compiler.
   This is due to an upstream change in llvm.  Use `-std=c++14` (or technically
   `-std=gnu++14`) to revert to the previous default.
+- Closure warnings are now controlled via the standard `-Wclosure` warning flags
+  rather than via a specific/custom `CLOSURE_WARNINGS` setting.  The old
+  setting continues to work but will issue a deprecation warning.
+  If you link with `-Werror` but you don't want closure warnings to be errors
+  you can add `-Wno-error=closure` or `-Wno-closure`.
```

Comment:
does `-w` also suppress closure warnings along with all the other warnings?

Reply:
Yes, closure warnings are not special at all so `-w` will work.